lstchain/datachecks/dl1_checker.py

```diff
@@ -90,15 +94,15 @@ def check_dl1(filenames, output_path, max_cores=4, create_pdf=False):
             raise FileNotFoundError
 
     # try to determine which trigger_type tag is more reliable for
-    # identifying interlaved pedestals. We check which one has
+    # identifying interleaved pedestals. We check which one has
     # more values == 32, which is the pedestal tag. The one called
     # "trigger_type" seems to be the TIB trigger type. The fastest way to do
     # this for the whole run seems to be using normal pytables:
     trig_tags = {'trigger_type': [], 'ucts_trigger_type': []}
     for filename in filenames:
         with tables.open_file(filename,
                               root_uep='/dl1/event/telescope/parameters') as f:
-            for name in trig_tags.keys():
+            for name in trig_tags:
```

Comment:
why is the `keys()` removed?

Reply:
I think it is not needed. The IDE suggested removing it.